packages/at_client/lib/src/service/notification_service.dart

```diff
@@ -101,6 +101,9 @@ abstract class NotificationService {
 
   /// Returns the status of the given notificationId
   Future<NotificationResult> getStatus(String notificationId);
+
+  /// Returns the [AtNotification] of the given notificationId
```

Comment:
Lets document various return types for various inputs and exceptions thrown.

Ex: 1. A notification with a given id exists
      2. When it does not exist
      3. Error in fetching the values etc...